src/components/Avatar/styled/text.ts

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components/native';
+import { BaseProps } from '../../types';
+import { COLOR_WHITE } from '../../../styles/colors';
+
+interface Props extends BaseProps {
+    size?: string;
+    style?: object;
```

Comment:
same ^^
No need to extend from BaseProps and no define the style prop.

Reply:
done